week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,20 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code ", "your");
 console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+//   It calls a function to concat 2 words passed as parameters (using a method concat())
```

Comment:
Yes!  I'd call these words strings, but that's fine, and just to show that you know what concatenation is I'd use a different word.  e.g. "join".

Reply:
yes definitely a better word